CHANGELOG.md

```diff
@@ -1,11 +1,13 @@
-## 2023-02-11
+## 2023-02-14
 | What's new | |
 | :--------- | :-: |
-| **Add a Variant right from Studio**, via the new 'Add Variant' button in your Graph Settings. | ![image](https://user-images.githubusercontent.com/14367451/218590889-d8aaa9d8-f11f-4c54-8b2d-18c880798e8a.png)
+| With the right permission, **easily distinguish which types are entities in Explorer with an `E` icon!**| ![image](https://user-images.githubusercontent.com/86634858/218832452-469b58cf-4806-4c7c-a326-1be10f125a9a.png)
```

Comment:
```suggestion
| **Easily distinguish which types are entities in Explorer with an `E` icon!**| ![image](https://user-images.githubusercontent.com/86634858/218832452-469b58cf-4806-4c7c-a326-1be10f125a9a.png)
```

Entities should be available for all roles

Reply:
oh really! even consumers!! yay!